src/sentry/static/sentry/app/components/resourceCard.jsx

```diff
@@ -3,15 +3,14 @@ import PropTypes from 'prop-types';
 import styled from 'react-emotion';
 
 import {analytics} from 'app/utils/analytics';
-import ConfigStore from 'app/stores/configStore';
 import ExternalLink from 'app/components/links/externalLink';
 import {Panel} from 'app/components/panels';
 
 export default class ResourceCard extends React.Component {
   static propTypes = {
     title: PropTypes.string.isRequired,
     link: PropTypes.string.isRequired,
-    imgUrl: PropTypes.string.isRequired,
+    imgUrl: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
```

Comment:
Why this type change? Also this makes it unsafe on line 28 below.

Reply:
Yeah.. In our tests we do this:

https://github.com/getsentry/sentry/blob/5dbc8befd07244fa788a39b1f9aa348493876ee6/jest.config.js#L14

which exports an object. So when tests run on this it recieves an object and fails the test due to the type error :( I spent a bit of time trying to figure out how to fix. I'll add a comment describing why we did it

Reply:
I'll try to fix this.